### DISCUSSION TITLE
Why is the total number of commits smaller?

### DISCUSSION THREAD

**Discussion by Xiaokang2022** [anuraghazra/github-readme-stats#3893] [Q&A] [answered]
When I get it using the link below:

```html
<img align="center" width="400" src="https://github-readme-stats.vercel.app/api?username=Xiaokang2022&theme=transparent&show_icons=true&hide_border=true&show=reviews&hide_title=true&hide=contribs" />
```

Here's what it looks like:

<img align="center" width="400" src="https://github-readme-stats.vercel.app/api?username=Xiaokang2022&theme=transparent&show_icons=true&hide_border=true&show=reviews&hide_title=true&hide=contribs" />

But when I add `&include_all_commits=true`, it looks like this:

<img align="center" width="400" src="https://github-readme-stats.vercel.app/api?username=Xiaokang2022&theme=transparent&show_icons=true&hide_border=true&show=reviews&hide_title=true&hide=contribs&include_all_commits=true" />

Is this number of commits correct? Why is the total number of commits smaller???

**Comment by ilianoKokoro** [comment, marked as answer]
Hello, @Xiaokang2022 . The reason the number of commit displayed is smaller than the actual one is that only public activity is counted for technical reasons. The app cannot access your private information without an GitHub token. Displaying private contributions require [deploying you own Vercel instance](https://github.com/anuraghazra/github-readme-stats#deploy-on-your-own) with your own GitHub API token.
